### ISSUE TITLE
Click on last point doesn't work

### ISSUE THREAD

**Issue by edomazi** [frappe/charts#258]
I have the following chart in which I'd like to be able to click the last point and get the information (value and label data) but I'm not able to. I need to click on it directly and not clicking firstly on another and then on the last one.
Am I missing something?

Chart demo: [https://codepen.io/edomazi/pen/zYxxjVy](https://codepen.io/edomazi/pen/zYxxjVy)

Frappé Charts version: 1.3.0

**Comment by scmmishra** [comment]
@edomazi 

> I'd like to be able to click the last point and get the information (value and label data) but I'm not able to.

Thanks for reporting this. Sure seems like something missing in the library. Just to avoid any confusion, from what I understand you want the event to be triggered automatically for the last item, without have the user to click it themselves?

**Comment by scmmishra** [comment]
> If you see on demo, when isNavigable is true, frappe select automatically last element.

Yes, this behaviour is intentionally added. Any reason you don't want it selected by default?

**Comment by edomazi** [comment]
@scmmishra

Thanks for answering. No, I'd like, when isNavigable is true, to have no item selected as default (now it automatically selects the last point as hammeronthenet said too). So if a user decides to click on the last point when the chart loads he would be able to and therefore I will do some parsing with the data point he just clicked on. 

To be clear, I'm able to get the values from the last point but only if firstly i click on another one then on the last one.

**Comment by scmmishra** [comment]
We can have a configuration, `navigateToLastItem` option, with default set to true.

How does this sound?

**Comment by edomazi** [comment]
That sounds great. So everyone is free to use this how they please

**Comment by hammeronthenet** [comment]
Yes, for example when I have two charts connected. The first present the
main data and the second (onclick) is the detail of the first one. If
someone clicks on the last value he is unable to select the data and the
only way is to click another "bar" and next click the wanted.

Il ven 26 giu 2020, 11:59 Shivam Mishra <notifications@github.com> ha
scritto:
